<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1211</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Credit">
     <attribute name="title">
      <string>Credit</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>23</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Общая сумма, rub:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>105</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Процентная ставка:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>473</x>
        <y>240</y>
        <width>34</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Тип:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="annuity">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>270</y>
        <width>133</width>
        <height>26</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Аннуитетный</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="differentiated">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>239</y>
        <width>212</width>
        <height>26</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </widget>
     <widget class="QComboBox" name="period">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>125</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Месяцев</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Лет</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>166</y>
        <width>46</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Срок:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="total_value">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="proc_stav">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="srok">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QPushButton" name="calculation">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>211</width>
        <height>38</height>
       </rect>
      </property>
      <property name="text">
       <string>Рассчет</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="credit_result">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>256</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Calculator">
     <attribute name="title">
      <string>Calculator</string>
     </attribute>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>541</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Close_br">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="MaxX">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>340</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>-1000000</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Umn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NumX">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Atan">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sum">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num9">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Result">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exp">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Okno">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>71</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="maxLength">
       <number>256</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>340</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X min:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Mod">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sqrt">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Del">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Log">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Tan">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>230</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cos">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sub">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>340</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X max:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ln">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="MinX">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>340</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>-1000000</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="Sin">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Point">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Asin">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Acos">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Open_Br">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>300</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AC">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Num0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
